.github/ruff.toml

```diff
@@ -0,0 +1,6 @@
+select = ["E"]
+ignore = ["E402", "E722", "E902"]
+exclude = ["*_pb2.py", "*_pb2.pyi", "capa2yara.py"]
```

Comment:
why would we want to exclude `capa2yara.py` here?

Reply:
capa2yara.py throws E501 Line too long for TODO.  so, I was just trying to see the log of tests whats breaking , by ignoring it just for tests.